### PR TITLE
TokuDB removal from package testing Job

### PR DESCRIPTION
**Pull request by kaushikpuneet07**
As TokuDB is deprecated from PS 8.0.28 onward , so we are going to remove tests that are meant for Tokudb as they are not needed now.

### CODE REVIEW

**Review by hrvojem — CHANGES_REQUESTED** [review]
why remove this? this task is installing percona-mysql-shell, nothing to do with TokuDB

**Comment by kaushikpuneet07** [comment]
@hrvoje , i am not removing mysql-shell , just checking something by removing it . I will add it for sure.

**Comment by kaushikpuneet07** [comment]
Hi @hrvojem and @eleo007 , i updated PS 8.0 Package testing install job , kindly review .

Till now we don't have mysql shell packages , so version script is giving error about incorrect mysql-shell version, thats why i disabled version script for now .

Sample run with update : https://ps80.cd.percona.com/job/package-testing-ps80/32/

File: playbooks/ps_80.yml [hunk 1]
```diff
@@ -92,34 +92,17 @@
...
-  - name: re-run ps-admin to finish tokudb install
-    command: /usr/bin/ps-admin --enable-tokudb
-
   - name: install percona-mysql-shell package
     include_tasks: ../tasks/install_pshell.yml
 
-  - name: check that Percona Server version is correct
-    command: /package-testing/version_check.sh ps80
+    #  - name: check that Percona Server version is correct
+    #    command: /package-testing/version_check.sh ps80
```
Comment (eleo007) [inline playbooks/ps_80.yml:105]: Why do we remove version check step?

File: tasks/test_prep.yml [hunk 2]
```diff
@@ -140,10 +140,10 @@
     with_items:
     - rm -rf /package-testing
     - rm -f master.zip
-    - wget --no-check-certificate -O master.zip https://github.com/Percona-QA/package-testing/archive/master.zip
+    - wget --no-check-certificate -O master.zip https://github.com/kaushikpuneet07/package-testing/archive/PS-8131.zip
```
Comment (eleo007) [inline tasks/test_prep.yml:143]: It seems this change should be reverted to Percona-QA.

File: playbooks/ps_80.yml (hunk 1 above)
Comment (kaushikpuneet07) [reply playbooks/ps_80.yml:105]: Percona-mysql-shell is not updated yet for PS 8.0.28 , so in version check step it is throwing error while checking percona-mysql-shell version , so it is commented out till build team uploads latest percona-mysql-shell packages

File: tasks/test_prep.yml (hunk 2 above)
Comment (kaushikpuneet07) [reply tasks/test_prep.yml:143]: Yeah , definitely i will revert it to  Percona-QA before merge

File: package_check.sh [hunk 3]
```diff
@@ -104,8 +104,8 @@ if [ ${product} = "ps56" -o ${product} = "ps57" -o ${product} = "ps80" ]; then
         rpm_opt_package="Percona-Server-tokudb-${rpm_maj_version} Percona-Server-rocksdb-${rpm_maj_version}"
       fi
     elif [ "${product}" = "ps80" ]; then
-        rpm_num_pkgs="9"
-        rpm_opt_package="percona-server-tokudb-${rpm_maj_version} percona-server-rocksdb-${rpm_maj_version} percona-server-shared-compat-${rpm_maj_version}"
```
[the remark this answers is not in the document]
Comment (kaushikpuneet07) [reply package_check.sh:108]: Packages names are still the same , it was a bug in package-check script , we don't have to match `percona-server-server-8.0 ` , package names for PS 8.0 are always like `percona-server-server`
8.0 is embeded by ${rpm_maj_version} and ${deb_maj_version} , so i removed them from 8.0 code.

File: playbooks/ps_80.yml (hunk 1 above)
Comment (kaushikpuneet07) [reply playbooks/ps_80.yml]: Done , executed one test for focal and it is also successful : https://ps80.cd.percona.com/job/package-testing-ps80-build/309/

Comment (kaushikpuneet07) [reply playbooks/ps_80.yml]: https://github.com/Percona-QA/package-testing/pull/147/commits/f699325459739a1c27814210a33c7359f41bea3b

**Comment by kaushikpuneet07** [comment]
Upgrade run after updating upgrade script

https://ps80.cd.percona.com/job/package-testing-ps80/37/

**Pull request merged**